src/boss/scaling.go

```diff
@@ -0,0 +1,63 @@
+package boss
```

Comment:
KJ will make that into another directory under boss dir. We think autoscaling should be under the boss dir because only boss will use that. Does that sound good to you?

Reply:
Great!  Yes, definitely under the boss dir.